nodl_python/test/nodl/_parsing/test_nodl_parse.py

```diff
@@ -40,14 +40,16 @@ def test__parse_element_tree(mocker):
 def test_parse_nodl_file_valid(mocker):
     mocker.patch('nodl._parsing._parsing._parse_element_tree')
 
+    path = Path(__file__).parents[1] / 'test.nodl.xml'
```

Comment:
We're doing this in a few places; let's write a pytest fixture for it in a `conftest.py` next to the data.

Reply:
Done in 63f04f2